### ISSUE TITLE
gp ports command

### ISSUE THREAD

**Issue by loujaybee** [gitpod-io/gitpod#9930]
Currently, in IDE's such as JetBrains, and when SSH'ing into Gitpod it's not (yet) possible to manage your Gitpod ports (e.g. list open ports). This issue is to introduce a `gp` CLI command which allows a user to modify and manipulate the ports of their workspace. Ideally, in the long run, the functionality available in each of the IDE's and `gp` CLI is mirrored / consistent.

**Comment by loujaybee** [comment]
Relates to: https://github.com/gitpod-io/gitpod/issues/9286

**Comment by akosyakov** [comment]
The first version should be 🛹 and allow following:
#### List ports
```
gp ports list
```

This command should print a table of ports with following columns:
- status: not served|failed to expose|exposing...|served
- label: port name + number
- visibility: public|private
- url
- description


#### Change visibility of ports
```
gp ports visibility <port>:{public|private}...
```

Example:
```
gp ports visibility 8000:public 3000:private
```

**Comment by loujaybee** [comment]
Issue for docs: https://github.com/gitpod-io/website/issues/2178

**Comment by akosyakov** [comment]
@felladrin is it resolved? 

I noticed that `watch gp ports` does not look nice. I think @andreafalzetti has some ideas how to improve.

**Comment by andreafalzetti** [comment]
I've created a dedicated issue for `watch gp ports` because it's a side effect of colors and they are also used in other commands, so I think we can fix all-in-one PR. 

Link to dedicated issue: https://github.com/gitpod-io/gitpod/issues/10633

**Comment by andreafalzetti** [comment]
I've raised https://github.com/gitpod-io/gitpod/pull/10638 to fix #10633 - which solves the output issue with `watch` for both commands `gp ports lists` and `gp tasks list`.

**Comment by akosyakov** [comment]
@loujaybee @felladrin I removed it from scheduled it for now. Since we have gp ports now and it is better to improve other aspects of JB integration.
